src/App.jsx

```diff
@@ -31,7 +32,7 @@ const App = () => {
           <Route path="/create-profile" component={CreateProfilePage}/>
           <Route path="/approved-clients" component={ApprovedClientsPage}/>
           <Route path="/onboarding-clients" component={() => <div>ONBOARDING CLIENTS</div>}/>
-          <Route path="/conversations/:clientID?" component={() => <div>CONVERSATIONS</div>}/>
+          <Route path="/conversations/" component={ConversationsPage}/>
```

Comment:
clientID param needs to be in the path so that when clicking on 'send a message' via the client list we are able to retrieve the client they are messaging